core/templates/base-components/loading-message.component.ts

```diff
@@ -24,7 +24,9 @@ import { downgradeComponent } from '@angular/upgrade/static';
    templateUrl: './loading-message.component.html'
  })
 export class LoadingMessageComponent {
-   @Input() message: string;
+  // This property is initialized using angular lifecycle hooks, see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1.
```

Comment:
Commented this on your other PR, please use the same comment here.

Reply:
done